web/src/styles/story.css

```
@@ -0,0 +1,296 @@
+:root{
+    --font-sans:'Inter',ui-sans-serif,system-ui,-apple-system,sans-serif;
+    --font-display:'Space Grotesk','Inter',ui-sans-serif,system-ui,sans-serif;
+    --font-mono:'JetBrains Mono',ui-monospace,'SF Mono',Menlo,Consolas,monospace;
```

Stylelint (17.14.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)